### PR TITLE
Rafactor: root package models

### PR DESCRIPTION
**Pull request by ben-mathu**
Part of an ongoing code refactoring to move all models to a common models package.

Fixes #4782

Moved main models to data.models package.

Tested debug on Pixel 4 (Emulator) with API level 30.

### CODE REVIEW

**Comment by Rishavgupta12345** [comment]
contributor review:-

why are you making so many pull requests, you are solving a single issue which is #4782 in parts so make a single branch and merge all your commits in one pull request, it's better than flooding the repository with tons of pull requests. Just keep the repository clean.

**Comment by ben-mathu** [comment]
> Thanks, @ben-mathu Please do make multiple PRs for this issue. Ideally one per package.

This is what I understood from your comment above in issue: #4782 [here](https://github.com/commons-app/apps-android-commons/issues/4782#issuecomment-1049059968)

**Comment by ben-mathu** [comment]
@madhurgupta10 please review

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,327 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
```
Comment (neslihanturan) [inline .idea/codeStyles/Project.xml:1]: this file shouldnt be edited

**Review by neslihanturan — CHANGES_REQUESTED** [review]
Thanks a lot @ben-mathu , there are irrelevant edits in this PR. You can always check files changes from github UI to make sure your PR has only needed changes. You can also use git status before adding your changes to your commits to prevent this. Please include only relevant changes.

Also there are empty .jpg files and they are not wanted.

**Comment by psh** [comment]
Closing: 2 years old and looks abandoned.

**Pull request closed without merging**